templates/extranode-floating.yaml

```diff
@@ -0,0 +1,31 @@
+heat_template_version: 2015-04-30
+
+# Template that creates a new floating IP to access the undercloud
+
+parameters:
+  external_net:
+    type: string
+    default: external
+    description: An external network from which floating ips can be provisioned
+
+  undercloud_port:
```

Comment:
The parameter and resource names in this file should be changed to not collide with the undercloud.  Also as far as I can tell this template is broken because the output references a resource that doesn't actually exist.